docs/user-manual/Readme.org

```diff
@@ -55,15 +48,42 @@ Select the first entry and press ~Enter~. This will boot the PlayOS installer. A
 #+attr_html: :width 800px
 [[../screenshots/install-playos.png]]
 
-To continue installation enter ~y~ and press ~Enter~. Note that installation will delete any data already present on the disk.
+To continue installation enter ~y~ and press ~Enter~. Note that installation will delete any data already present on the computer's storage disk.
+
+Installation takes around 5 to 10 minutes. After the installation completes successfully, the computer will automatically restart the computer and boot into the newly installed PlayOS system.
+
+** Required boot configuration in BIOS
```

Comment:
It is only a very small and relatively stable group of people who install PlayOS on a machine that has been never set up with PlayOS. Most cases would be reinstalling a machine to switch channel, try a lab environment, etc. My thinking was hence that for the majority it would be more confusing than helpful to start with the instructions for BIOS configuration.

Reply:
I reordered the first paragraph: https://github.com/dividat/playos/pull/113/files#r1108117926